libs/features/dashboard/mentor/src/lib/features-dashboard-mentor.tsx

```diff
@@ -1,16 +1,59 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { createStructuredSelector } from 'reselect';
+import { advertsActions, advertsSelectors } from '@ppm/data-access/adverts';
+import { SharedAdvertCard } from '@ppm/shared/advert-card';
 
 import './features-dashboard-mentor.scss';
 
 /* eslint-disable-next-line */
 export interface FeaturesDashboardMentorProps {}
 
+const stateSelector = createStructuredSelector({
+  adverts: advertsSelectors.selectAdverts(),
+  loading: advertsSelectors.selectLoading(),
+});
+
 export const FeaturesDashboardMentor = (
   props: FeaturesDashboardMentorProps
 ) => {
+  const dispatch = useDispatch();
+  const { adverts, loading } = useSelector(stateSelector);
+
+  useEffect(() => {
+    // TODO take ID from state
+    dispatch(advertsActions.getAllByAuthor('5f2d0f4c022c954774d5313e'));
```

Comment:
if you will get logged user all adverts, you don't need have id, because BE already know this id if user is logged in